### ISSUE TITLE
Question: Should NavigationViewItem work on 15063 like on later Windows versions?

### ISSUE THREAD

**Issue by hihain** [microsoft/microsoft-ui-xaml#2382]
I thought that I should use `NavigationViewItem `inside my `MenuItemTemplate`, but that does not seem to work on 15063. Replacing it with a `TextBock` solves the difference in style and function between 15063 and newer versions of Windows. I use the current stable UI.XAML 2.3 release.

**Is it a bug or am I misunderstanding how to bind data to a `NavigationView`?**

My problem is, that it works flawlessly on 16299+, whereas 15063 has weird displaying issues. The biggest problem using 15063 is having a very small click area that is not where the content text is displayed, but a bit on the left side. I marked the spot with a red arrow in the image comparison.

Examples:

![works](https://user-images.githubusercontent.com/1473059/80997088-2439af00-8e41-11ea-9602-994ac65ff7d7.PNG)
```
<muxc:NavigationView x:Name="works" 
                                  MenuItemsSource="{x:Bind Categories, Mode=OneWay}" 
                                  PaneDisplayMode="Top" 
                                  IsBackEnabled="False" IsSettingsVisible="False" IsBackButtonVisible="Collapsed">
            <muxc:NavigationView.MenuItemTemplate>
                <DataTemplate x:DataType="local:Category">
                    <TextBlock Text="{x:Bind Name}" />
                </DataTemplate>
            </muxc:NavigationView.MenuItemTemplate>
        </muxc:NavigationView>
```

![clickPoint](https://user-images.githubusercontent.com/1473059/80997094-256adc00-8e41-11ea-8a0f-45a6be29118d.PNG)
```
<muxc:NavigationView x:Name="strangeOn15063" 
                                  MenuItemsSource="{x:Bind Categories, Mode=OneWay}" 
                                  PaneDisplayMode="Top" 
                                  IsBackEnabled="False" IsSettingsVisible="False" IsBackButtonVisible="Collapsed">
            <muxc:NavigationView.MenuItemTemplate>
                <DataTemplate x:DataType="local:Category">
                    <muxc:NavigationViewItem Content="{x:Bind Name}" />
                </DataTemplate>
            </muxc:NavigationView.MenuItemTemplate>
        </muxc:NavigationView>
```

**Comment by ranjeshj** [comment]
@hihain can you try the latest 2.4 pre-release build ? There have been several changes to the NavigationView control since 2.3.

**Comment by hihain** [comment]
Amazing! I switched to the today released 2.4 version and it works great!

BTW: How would you solve the challenge to support 14393? UI.XAML 2.1 does has the same problems as 2.2 and 2.3 (described above).
